run_unittests.py

```diff
@@ -1865,6 +1865,21 @@ def test_bad_option_subproject(self):
             out,
             r'In subproject one: Unknown command line options: "one:two"')
 
+    def test_objc_cpp_detection(self):
+        '''
+        Test that when we can't detect objc or objcpp, we fail gracefully.
+        '''
+        env = Environment('', self.builddir, self.meson_command,
+                          get_fake_options(self.prefix), [])
+        try:
+            objc = env.detect_objc_compiler(False)
```

Comment:
__[Flake8]__

[F841] local variable 'objc' is assigned to but never used


<!-- Comment by [SideCI](https://sideci.com) -->